### PR TITLE
More registrations work

### PR DESCRIPTION
**Pull request by markreesmoj**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt [hunk 1]
```diff
@@ -1,61 +0,0 @@
...
-      assertThat(result.points).isEqualTo(10)
-    }
-  }
-
-  @Nested
-  @DisplayName("Simple RSR tests")
-  inner class SimpleRSRTests {
-
-    @Test
-    fun `should return 0 for RSR null`() {
```
Comment (markreesmoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt:50]: RSR cannot be null

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt [hunk 2]
```diff
@@ -1,61 +0,0 @@
...
-      listOf(),
-      0
-    )
-  }
-
-  @Nested
-  @DisplayName("Simple Risk tests")
-  inner class SimpleRiskTests {
-
-    @Test
```
Comment (markreesmoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt:37]: Covered by BDD

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt [hunk 3]
```diff
@@ -1,61 +0,0 @@
...
-  @DisplayName("Simple RSR tests")
-  inner class SimpleRSRTests {
-
-    @Test
-    fun `should return 0 for RSR null`() {
-      val result = calculateProtectLevel(rsr = BigDecimal.ZERO)
-      assertThat(result.points).isEqualTo(0)
-    }
-
-    @Test
```
Comment (markreesmoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ProtectLevelCalculatorTest.kt:55]: Covered by BDD

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ChangeLevelCalculatorTest.kt [hunk 4]
```diff
@@ -51,7 +51,7 @@ internal class ChangeLevelCalculatorTest {
     fun `should calculate Oasys Needs none`() {
       val assessment = OffenderAssessment("12345", LocalDateTime.now(clock), null, "AnyStatus")
 
-      val result = service.calculateChangeLevel(crn, assessment, 0, listOf(), getValidConviction(), mapOf())
+      val result = service.calculateChangeLevel(crn, assessment, 0, false, getValidConviction(), mapOf())
```
Comment (markreesmoj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/ChangeLevelCalculatorTest.kt:54]: This is a bit of a code smell - it's always false in this test

**Review by nickmcmahon01 — APPROVED** [review]

**Pull request merged**